packages/components/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,30 @@
+import { TabsProps, TabsStyledProps } from './tabs.types';
+import { TabStyled } from './tabs.styles';
+
+const Tab = ({ 
+    label, 
+    value, 
+    selected, 
+    disabled, 
+    onClick 
+}: TabsProps & 
+TabsStyledProps & 
+{ onClick: (value: string) => void }) => {
+  const handleClick = () => {
+    if (!disabled) {
+      onClick(value);
+    }
+  };
+
+  return (
+    <TabStyled 
+        className={selected ? 'active' : ''} 
```

Comment:
Check the example in this tutorial.
Managing conditional styles is more easy than this.